src/compiler/AST/Frontend.elm

```diff
@@ -31,6 +31,7 @@ type Expr
     | Call { fn : Expr, argument : Expr }
     | If { test : Expr, then_ : Expr, else_ : Expr }
     | Let { bindings : List (Binding Expr), body : Expr }
+    | List { list: List Expr }
```

Comment:
What is the advantage of using a record here?

Reply:
it might have been copying the style of constructors above.

my original intention with those was "if one argument, keep it naked, if it's eg. 3x Expr, name them to prevent confusion"

I'd unwrap the record in this case

Reply:
I'm going to remove this code, it's out of the scope of this PR. It was more like an exercise.